andela/titan-dev-ah-frontend
# #163518636 Setup Redux

#### What does this PR do?
- Setup redux 
#### Description of Task to be completed?
- Include redux in the application
- Connect react with redux 
- Include redux-thunk in the application
- Create dummy actionCreator, store, and reducer 
- Ensure that dummy actionCreator, store, and reducer is working
#### How should this be manually tested?
- Fork the repo and check out the branch `ch-setup-redux-163518636`
- In the root folder run `yarn` to install all dependencies
- Run `yarn start` to start the project
- The browser should be opened. Click the button label `Fetch more articles`. 
- You should see more articles being added to the page. 

#### What are the relevant pivotal tracker stories?
#163518636
